site/content/entry/yuzu-progress-report-sep-2023/index.md

```diff
@@ -0,0 +1,379 @@
+    "./tt1.png"
+    "./tt2.png"
+    "./tt3.png"
+    >}}
+
+Back again with good fixes is [Squall-Leonhart](https://github.com/Squall-Leonhart), who now focused on Windows file system error handling.
+For example, yuzu used to just crash if you disconnected a removable storage containing games loaded into yuzu’s list, or a network SMB share was disconnected, or a Bitlocker protected location was unavailable.
+
+But not anymore! By adding the {{< gh-hovercard "11499" "proper error handling" >}} to these file system exception cases, the emulator now avoids a crash here; complaining, but working. 
+
+Managing another critical blow to the fight for proper shutdown behaviour, byte[] managed to spot an {{< gh-hovercard "11652" "optimization" >}} that should save around a second of time on high-end CPUs.
+Does this mean the fighting is over? No, some edge cases remain, but now under normal circumstances, closing a game should be almost instantaneous.
+We have a medal ready for byte[].
+
+## Hardware section
+
+### NVIDIA, VRAM fixes, and focusing on the latest hardware
+
+The latest driver releases have proven to be stable for Turing and newer products, but we got reports of small regressions affecting Pascal and Maxwell users.
+If you’re still running older 750, 900 and 1000 series products and you face new issues in games, try reverting to older driver versions like the 52X series.
+
+Still, it's worth to mention that driver release `545.84` seems to have solved the instabilities caused by excessive VRAM use, like for example while running `The Legend of Zelda: Tears of the Kingdom` in the Depths with 2GB/4GB GPUs of any architecture.
+This is great news for low end users struggling to run this particularly ASTC-heavy game.
+
+### AMD, fixes, AutoHDR, VSync problems and frame interpolation
+
+First, good news, latest Windows driver version at the time of writing, `23.10.2` has solved the D24 issue affecting AMD cards on Vulkan.
+Those bothered by clipping lines and textures should upgrade to get the proper graphics experience.
+Keep in mind that this doesn't solve the vertex explosions affecting the Pentelas region and the DLC of `Xenoblade Chronicles 3`, we manage to confirm this is a different issue affecting AMD cards ona much lower level, as it is architecture, OS, and driver agnostic.
+
+Another good bit is that recent drivers also seem to have introduced DXGI swapchain support, allowing AMD cards to work with AutoHDR out of the box. 
+There doesn't seem to be a driver toggle for this so
+
+Now for a bit of bad news. 
+Windows driver version `23.9.3` and newer cause the VSync options to be ignored, resulting in tearing during gameplay.
+If you’re affected by this, revert back to driver 23.9.2 for now.
+The standard procedure was followed: we opened a ticket of this issue to AMD with a generic test case.
+
+Switching to something more interesting, we heard you! Despite our previous (deserved) rants about frame generation, we think
```

Comment:
```suggestion
Switching to something more interesting, we heard you! Despite our previous rants about frame generation, we think
```
also link to the relevant page where the thing was (https://yuzu-emu.org/entry/yuzu-progress-report-may-2023/)